inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/hive/HiveSinkRequest.java

```diff
@@ -41,7 +41,6 @@
 @JsonTypeDefine(value = SinkType.HIVE)
 public class HiveSinkRequest extends SinkRequest {
 
-    @NotBlank(message = "jdbcUrl cannot be blank")
     @ApiModelProperty("Hive JDBC URL, such as jdbc:hive2://${ip}:${port}")
```

Comment:
Maybe need more UT to ensure the blacnk jdbcUrl will not cause some unexcepted failure.